src/Forms/RemoveTextFromHearImpaired.cs

```diff
@@ -76,32 +76,24 @@ private void FixLargeFonts()
 
         public static string RemoveStartEndNoise(string text)
         {
-            string s = text.Trim();
-            if (s.StartsWith("<b>") && s.Length > 3)
-                s = s.Substring(3);
-            if (s.StartsWith("<i>") && s.Length > 3)
-                s = s.Substring(3);
-            if (s.StartsWith("<u>") && s.Length > 3)
-                s = s.Substring(3);
-            if (s.StartsWith("<B>") && s.Length > 3)
-                s = s.Substring(3);
-            if (s.StartsWith("<I>") && s.Length > 3)
-                s = s.Substring(3);
-            if (s.StartsWith("<U>") && s.Length > 3)
-                s = s.Substring(3);
-
-            if (s.EndsWith("</b>") && s.Length > 4)
-                s = s.Substring(0, s.Length - 4);
-            if (s.EndsWith("</i>") && s.Length > 4)
-                s = s.Substring(0, s.Length - 4);
-            if (s.EndsWith("</u>") && s.Length > 4)
-                s = s.Substring(0, s.Length - 4);
-            if (s.EndsWith("</B>") && s.Length > 4)
-                s = s.Substring(0, s.Length - 4);
-            if (s.EndsWith("</I>") && s.Length > 4)
-                s = s.Substring(0, s.Length - 4);
-            if (s.EndsWith("</U>") && s.Length > 4)
+            const int i = ('i' | 'I');
+            const int b = ('b' | 'B');
+            const int u = ('u' | 'U');
+
+            var s = text.Trim();
+            // <i> <I> | <b> <B> | <u> <U>
+            if (s.Length > 3 && s[0] == '<' && s[2] == '>' &&
+                ((s[1] & i) == text[1] || (s[1] & b) == s[1] || (s[1] & u) == s[1]))
+            {

```

Comment:
yeah I know but since the subrip only use <i>, <p>, <u>... I think it wouldn't make difference that is why I check for '<' && '>' first